projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -3947,7 +3947,13 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
             }
             const row = new IgxRow(rowSelector, -1, this.gridAPI.getRowData(rowSelector));
             this.gridAPI.update_row(row, value);
-            this.cdr.detectChanges();
+
+            // TODO: fix for #5934 and probably break for #5763
+            // consider adding of third optional boolean parameter in updateRow.
+            // If developer set this parameter to true we should call notifyChanges(true), and
+            // vise-versa if developer set it to false we should call notifyChanges(false).
+            // The parameter should default to false
+            this.notifyChanges();
```

Comment:
Would UpdateCell benefit from this change as well?  I haven't tested specifically with that but see the detectChanges is being used there as well.

Reply:
@demingjs 
No, definitely no. This will break issue #5763.
There is a case when you may need to update a cell after another cell is changed, e.g. you have row with price, quantity and total. If user changes the value of let say quantity you would want to change the value of total by calling `updateCell`. If we do not call `detectChanges` at the end of `updateCell` total cell will not display updated value.
To summarize - if you need to update quickly the values of many cell, as in your case, you should call `updateRow`. If you need to change single cell you should call `updateCell`.